ephios/core/views/settings.py

```diff
@@ -46,4 +54,30 @@ def form_valid(self, form):
         return super().form_valid(form)
 
     def get_success_url(self):
-        return reverse("core:settings_general")
+        return reverse("core:settings_instance")
+
+
+class NotificationSettingsView(
+    LoginRequiredMixin, SuccessMessageMixin, SettingsViewMixin, FormView
+):
+    template_name = "core/settings/settings_notifications.html"
+    success_message = _("Settings succesfully saved.")
+
+    def get_form(self, form_class=None):
+        return UserNotificationPreferenceForm(self.request.POST or None, user=self.request.user)
+
+    def get_success_url(self):
+        return reverse("core:settings_notifications")
+
+    def form_valid(self, form):
+        form.update_preferences()
+        return super().form_valid(form)
+
+
+class PasswordChangeSettingsView(PasswordChangeView, SettingsViewMixin):
+    template_name = "core/settings/password_change_form.html"
+    success_url = reverse_lazy("core:settings_password_change_done")
```

Comment:
![image](https://user-images.githubusercontent.com/5970076/179397328-3bcffd16-21d7-4040-af40-9e203315a5aa.png)

The management menu entries are missing in this view.

Reply:
wrong inheritance order